backend/src/apiserver/common/config.go

```diff
@@ -35,6 +35,7 @@ const (
 	MetadataGrpcServiceServiceHost          string = "METADATA_GRPC_SERVICE_SERVICE_HOST"
 	MetadataGrpcServiceServicePort          string = "METADATA_GRPC_SERVICE_SERVICE_PORT"
 	SignedURLExpiryTimeSeconds              string = "SIGNED_URL_EXPIRY_TIME_SECONDS"
+	MLPipelineTLSEnabled                    string = "ML_PIPELINE_TLS_ENABLED"
```

Comment:
@hbelmiro I'm a little confused by the naming of this `MLPipelineTLSEnabled`, it's saying that mlpipeline is tls enabled, mlpipeline in the code/kfp typically refers to the kfp server, not mlmd server, but this variable looks like it's being used to determine if it's infact mlmd that's tls enabled, i.e. mlmd is accepting tls connections, is that correct? If so maybe this should be caled `MLMDTLSEnabled` ?

Reply:
Same as https://github.com/opendatahub-io/data-science-pipelines/pull/72#discussion_r1747684691.

Reply:
Clarified offline.
I added a new flag for MLMD.